code/modules/events/spacevine.dm

```diff
@@ -146,7 +146,13 @@
 	quality = NEGATIVE
 
 /datum/spacevine_mutation/aggressive_spread/on_spread(obj/structure/spacevine/holder, turf/target)
-	target.ex_act(severity, null, src) // vine immunity handled at /mob/ex_act
```

Comment:
This change makes aggressive spreading space vines explode turfs and deal friendly fire/kill vine immune mobs. I later fixed that on TG with this PR:
https://github.com/tgstation/tgstation/pull/52650

Reply:
Fixed thank you for the heads up.